package.json

```diff
@@ -49,7 +49,7 @@
     "pretest": "jscs --esprima=esprima-fb ./",
     "test": "karma start --single-run",
     "coveralls": "cat tests/coverage/*/lcov.info | node_modules/coveralls/bin/coveralls.js",
-    "build": "webpack -p",
+    "build": "webpack -p --watch",
```

Comment:
there's no need to `watch` for a `build`. You can add a different script for this.
`"watch": "npm run build -- --watch"`